config/prow/plugins.yaml

```diff
@@ -321,21 +321,21 @@ slack:
 
 milestone_applier:
   kubernetes/enhancements:
-    master: v1.19
+    master: v1.20
   kubernetes/kubernetes:
-    master: v1.19
+    master: v1.20
     release-1.19: v1.19
     release-1.18: v1.18
     release-1.17: v1.17
     release-1.16: v1.16
   kubernetes/release:
-    master: v1.19
+    master: v1.20
   kubernetes/sig-release:
-    master: v1.19
+    master: v1.20
   kubernetes/test-infra:
-    master: v1.19
+    master: v1.20
   kubernetes/kops:
-    master: v1.19
+    master: v1.20
```

Comment:
Should kops be bumped to 1.20 already?

Reply:
Agreed. Please only bump release-related repos.

Reply:
too many 1.19 :p 
removed